doc/guides/collaborator-guide.md

```diff
@@ -95,9 +96,8 @@ request must pass code review and CI before landing into the codebase.
 
 ### Code Reviews
 
-At least two Collaborators must approve a pull request before the pull request
-lands. One Collaborator approval is enough if the pull request has been open
-for more than seven days.
+At least one Collaborator must approve a pull request before the pull request
+lands. Waiting for a second approval is recommended but not mandatory.
```

Comment:
```suggestion
lands. Waiting for a second approval is preferable.
```

Reply:
As discussed in the comment thread, I would prefer that we keep this at two collaborators to sign off within a 48 or 72 hour window. Beyond that, one is sufficient.

Reply:
Now that we have `CODEOWNERS` in place, can we say that if there is a CODEOWNER for a given change, then it must be one signoff from a CODEOWNER? Otherwise, any one signoff is ok?

Reply:
I'm fine either way (could also be a follow up PR), if we decide to require signoff from a CODEOWNER of each team added to the PR (note more teams can own a PR), we should:

1. Have automation in place to update the list of `CODEOWNERS` if the PR changes
2. Have automation in place to detect approvals and ✅ the appropriate teams on the bot comment
    1. We'll probably need to require using the Approve GitHub button instead of just a LGTM
3. Have ncu validate the reviewers as valid codeowners for all the files that needed codeowners sign-off

Seems like a lot, I think we can explore this idea but it shouldn't block this PR.

As a side note, I was already planning on implementing 1 and 2 anyway after moving the codeowners code to an Action :)

Reply:
Any additional tooling definitely shouldn't block this PR. I'd just like to make sure that if there is a codeowner, we're giving them the opportunity to review before something lands.